build-release.sh

```diff
@@ -1,21 +1,20 @@
 #!/bin/sh
+DIR=`dirname $(readlink -f $0)`
 
 version=${1:-centos5}
 
 case $version in
   centos5|centos)
-    ln -sf Dockerfile.centos5 Dockerfile
-    docker build -t centos5-smell-baron . || exit 1
```

Comment:
I think `-t $version-smell-baron` might be better, this way the same system can be used to test all versions without each subsequent docker container clobbering the previous one.